docs/Respo-Agent.md

```
@@ -0,0 +1,822 @@
+# Review the definition in detail
+cr query def namespace/definition
+```
+
+---
+
+## Common Patterns and Best Practices
+
+### 1. Component Definition Pattern
+
+**Cirru (Read):**
+```cirru
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)